eppy/tests/test_function_helpers.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pytest
+from six import StringIO
+
+import eppy.function_helpers as fh
+from eppy.bunch_subclass import register_epbunch_function
+from eppy.iddcurrent import iddcurrent
+from eppy.modeleditor import IDF
+from eppy.tests.test_bunch_subclass import idftxt
+
+
+class TestRegisterFunction:
+    @pytest.fixture()
+    def IDF(self):
+        iddfhandle = StringIO(iddcurrent.iddtxt)
+
+        if IDF.getiddname() == None:
+            IDF.setiddname(iddfhandle)
+        yield IDF
+
+    @pytest.fixture()
```

Comment:
I'd prefer these two fixtures to be in `conftest.py`. In fact I think there may be equivalent ones there already.

Reply:
corrected in 3080f0d607266f6bae0554640f89b0740071b1aa